README.md

```diff
@@ -43,11 +43,10 @@ Debian-based systems we recommend to use
 
 ### From source
 
-You can clone this directory and do a `make install`
+You can clone this directory and use `pip install .` (with `--user`, `-e`
```

Comment:
this won't include `dcmstack`, perhaps `pip install -r requirements.txt`?

Reply:
THANKS!  Added `dcmstack` into requires and thus would be included if `pip install .[all]`. Adjusted README to recommend ppl use `[all]` flavor. See 7f00b1a54ef38ae3cc7cc4cd059e10331ba97654